### DISCUSSION TITLE
Typescript should enforce a common eslint rule such as Airbnb TS

### DISCUSSION THREAD

**Discussion by jt6677** [realworld-apps/realworld#757] [Ideas]
I appreciate this repo. The inconsistency of typescript eslint rules caused me a lot of confusion. I think a real realworld should follow industry standard and apply some eslint rules such as Airbnb TS.

**Comment by geromegrignon** [comment]
Hey @jt6677, 
Thanks for your interest in this project.
This repo (https://github.com/gothinkster/realworld) doesn't use Typescript.
Is your request related to an implementation repository?

I agree about such standards, our latest node implementation uses Airbnb eslint rules: https://github.com/gothinkster/node-express-prisma-v1-official-app/blob/main/.eslintrc.json

**Reply by jt6677** [reply]
hey geromegrignon, I was referring to the  React implementation.  I recognize it might be too much to ask for.

**Reply by geromegrignon** [reply]
If it's about https://github.com/gothinkster/react-redux-realworld-example-app, I archived it as there were no maintenance for years and we received several submissions of community-driven React implementations.

If it's about one of the implementations listed on CodebaShow (https://codebase.show/projects/realworld), I encourage you to open an issue there to propose such change as we don't maintain those projects.

About standards themselves, I"m currently working on a new React Typescript implementation as the new frontend used for the demo and Airbnb eslint rules is definitely part of it.

**Reply by jt6677** [reply]
thank you for your reply. I am looking forward to your new demo!
